pydantic_ai_slim/pydantic_ai/_agent_graph.py

```diff
@@ -1129,6 +1144,7 @@ async def _call_tool(
                 tool_name=tool_call.tool_name,
                 content=tool_call_result.message,
```

Comment:
Similar to above; it'd be nice if we could move the default `'The tool call was denied.'` message off of `ToolDenied`, and only have it on `PromptTemplates`